Added new endpoint to tak in CSV file and then call the response serv…

sdiaz1000 opened the pull request:
…ice endpoint to create new users within the data base

ChrisHardmanONS committed:
Added new endpoint to tak in CSV file and then call the response service endpoint to create new users within the data base

ChrisHardmanONS committed:
made changes based on code review.

ChrisHardmanONS committed:
FWMT-289: updated endpoint to use correct column name

sdiaz1000 committed:
FWMT-289 Code review fixes

sdiaz1000 committed:
Merge branch 'FWMT-289' of
https://github.com/ONSdigital/fwmt-staff-service.git into FWMT-289

Conflicts:
	src/main/java/uk/gov/ons/fwmt/staff_service/controller/WebDriverController.java

garethTurner commented on src/main/java/uk/gov/ons/fwmt/staff_service/data/dto/UserDTO.java:
```diff
@@ -0,0 +1,18 @@
+
+    public String authNo;
+    public String tmUsername;
+    public Boolean activeDate;
```
Should just be active

JacobHarrisonONS commented on src/main/java/uk/gov/ons/fwmt/staff_service/controller/StaffController.java:
```diff
@@ -0,0 +1,52 @@
+  @RequestMapping(method = RequestMethod.POST,value = "/addUsersFromCSV", produces = "application/json", consumes = "multipart/form-data")
+  public void AddUsersFromCSV(@RequestParam("file") MultipartFile file) throws IOException {
+
+    Reader reader = new InputStreamReader(file.getInputStream());
```
extract this into a separate method in a service so it can be tested

ChrisHardmanONS committed:
FWMT-289: added new two new columns to be read from csv and also checks active status

garethTurner commented on src/main/resources/wsdl/Messaging_MessageQueueWs.wsdl:
```diff
@@ -590,7 +590,7 @@
       <!--<s:complexType name="ArrayOfUserType">-->
         <!--<s:sequence>-->
-          <!--<s:element minOccurs="0" maxOccurs="unbounded" name="User" type="s1:UserType" />-->
+          <!--<s:element minOccurs="0" maxOccurs="unbounded" name="UserDTO" type="s1:UserType" />-->
```
Dont think these should have been changed

ChrisHardmanONS committed:
fixed changes made to wsdl file replacing userDTO back to user

ChrisHardmanONS committed:
corrected if statment for strings

ChrisHardmanONS committed:
Merge branch 'master' into FWMT-289

Commit:
Reformatted all code
cleaned uo codacy issues:
a method stating with a captial
explicit scoping
unused imports
